src/screens/Profile/Header/Metrics.tsx

```diff
@@ -28,16 +32,21 @@ export function ProfileHeaderMetrics({
     one: 'following',
     other: 'following',
   })
+  const pluralizedPostCount = plural(profile.postsCount || 0, {
+    one: 'post',
+    other: 'posts',
+  })
 
   return (
     <View
-      style={[a.flex_row, a.gap_sm, a.align_center, a.pb_md]}
+      style={[a.flex_row, a.gap_sm, a.align_center, a.pb_md, {zIndex: 1}]}
```

Comment:
zIndex:1 was required to make the tooltip appear on top of the bio text below, idk if this is the "correct" place to put it

Reply:
```suggestion
      style={[a.flex_row, a.gap_sm, a.align_center, a.pb_md, a.z_10]}
```